tests/kubernetes/downtime/health.proto

```
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package grpc.health.v1;
```

Buf (1.47.2)

Files with package "grpc.health.v1" must be within a directory "grpc/health/v1" relative to root but were in directory "tests/kubernetes/downtime".

(PACKAGE_DIRECTORY_MATCH)